functional/FunctionalTest.java

```diff
@@ -211,6 +211,10 @@ public static void writeObject(String urlString, byte[] dataBytes) throws Except
     Request request = requestBuilder
         .url(HttpUrl.parse(urlString))
         .method("PUT", RequestBody.create(null, dataBytes))
+        // Set header 'x-amz-acl' to 'bucket-owner-full-control', so objects created
+        // anonymously, can be downloaded by bucket owner.
+        // Source: http://prettyplease.me/anonymous-s3-upload-with-full-owner-control/
+        .addHeader("x-amz-acl","bucket-owner-full-control")
```

Comment:
can you add a space after `.addHeader("x-amz-acl", "bucket-owner-full-control")`